README.md

```diff
@@ -23,27 +23,50 @@ If you'd like to help out, please see [CONTRIBUTING.md](CONTRIBUTING.md).
 * Future release
   - Planned: removal of unused signature verification schemes at link-time.
   - Planned: removal of unused protocol versions at link-time.
-* Next release: 0.21.0 (2023-xx-xx)
+* Next release: 0.21.0 (2023-03-29)
   - Support for connecting to peers named with IP addresses.  This means
     rustls now depends on a fork of webpki - `rustls-webpki` - with a suitably
     extended API.
   - *Breaking change*: `StoresClientSessions` trait renamed to `ClientSessionStore` and
     reworked to allow storage of multiple TLS1.3 tickets and avoid reuse of them.
     This is a privacy improvement, see RFC8446 appendix C.4.
-  - *Breaking change*: `rustls::Error` is no longer `PartialEq`.
+  - *Breaking change*: the `DistinguishedNames` type alias no longer exists; the public
+    API now exports a `DistinguishedName` type, and the
+    `ClientCertVerifier::client_auth_root_subjects()` method now returns a
+    `&[DistinctedName]` instead (with the lifetime constrained to the verifier's).
```

Comment:
`DistinctedName` -> `DistinguishedName`... I guess I wrote this?

Reply:
Good catch, fixed!